src/no-wallaby-comment.spec.ts

```diff
@@ -183,6 +183,20 @@ const LINE_11_WITH_COMMENTS_EXPECTED = `
  **/
 `;
 
+const LINE_12_WITH_COMMENTS = `
```

Comment:
I think we need to test `//` comments as well, both standalone and same line after a statement